book/fault_proofs/quick_start.md

```diff
@@ -31,9 +31,13 @@ This guide provides the fastest path to try out OP Succinct fault dispute games
     DISPUTE_GAME_FINALITY_DELAY_SECONDS=604800
     MAX_CHALLENGE_DURATION=604800
     MAX_PROVE_DURATION=86400
-    STARTING_ROOT=0xbd4ab027af3c4db0c6b1329dad6d9f8e2505accabec75bfaa2b6b8033c1e60c5...
+    STARTING_ROOT=0xbd4ab027af3c4db0c6b1329dad6d9f8e2505accabec75bfaa2b6b8033c1e60c5
     STARTING_L2_BLOCK_NUMBER=791000
 
+    # Optional
+    # For production setting, set PROPOSER_ADDRESSES and CHALLENGER_ADDRESSES properly.
```

Comment:
nit: could use more clarification ex. "Warning: setting this to true allows anyone to _. In a production setting, we recommend setting this to false and configuring PROPOSER_ADDRESSES and CHALLENGER_ADDRESSES accordingly."